app/src/main/java/com/codesoom/assignment/application/ProductService.java

```diff
@@ -36,6 +53,14 @@ public Product createProduct(ProductData productData) {
         return productRepository.save(product);
     }
 
+    /**
+     * 식별 값으로 상품을 조회하고,
+     * 상품정보로 상품을 수정하고,
+     * 수정 된 상품을 리턴합니다.
+     * @param id 식별 값
+     * @param productData 수정 할 상품정보
+     * @return 수정 된 상품
+     */
```

Comment:
`id`의 뜻과 의미는 상식이니까, 생략해도 괜찮아요.

```suggestion
    /**
     * 주어진 상품정보로 상품을 수정한 다음, 수정된 상품을 리턴합니다.
     * @param id 상품 식별값
     * @param productData 수정할 상품정보
     * @return 수정된 상품
     */
```